deploy/appspec.yml

```
@@ -0,0 +1,18 @@
+
+permissions:
+  - object: /
+    pattern: "**"
+    owner: ec2-user
+    group: ec2-user
+
+hooks:
+  ApplicationStart:
+    - location: deploy.sh
+      timeout: 60
+      runas: ec2-user
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)